.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Release
+about: When ready to cut a release
+title: Release X.Y.Z
+labels: release
+assignees: ''
+
+---
+
+- [ ] Start a new release branch:
+```bash
+$ git flow release start X.Y.Z
+```
+- [ ] Rotate `CHANGELOG.md` (following [Keep a Changelog](https://keepachangelog.com/) principles)
+- [ ] Ensure outstanding changes are committed:
+```bash
+$ git add .
```

Comment:
`git add .` can bring in files that you may have lingering in your working copy that you do not want to commit. We should consider using the more explicit `git add CHANGELOG.md`.

The OTM release checklist prescribes running the `git describe --always --dirty | grep -B1 dirty` command to check that you have a clean working directory. It might be good to include that at the top of this checklist too.

Reply:
I like the idea of making the reference to `$ git add ...` more explicit.

What do you think about running `$ git status` instead? It will also show if the staging area is dirty, but is more descriptive in the case that there are other last-minute fixes you'd like to lump into this commit.

Reply:
`git status` is good. The "dirty" command just gives you a one line yes/no are there changes. In this context it might be good to know what has changed.